.github/workflows/pytest-gpu.yml

```diff
@@ -70,11 +70,17 @@ jobs:
       with:
         command: |
           #! /bin/bash
-          # Ensure clang and cuda libraries are visible
+          # Ensure clang libraries are visible
           export PATH=/usr/local/cuda-10.1/bin:$PATH
+          # Ensure CUDA libraries are visible
           export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64:$LD_LIBRARY_PATH
           export PATH=~/install/bin:$PATH
```

Comment:
is this ~/install/bin really for CUDA?

Reply:
No. How did that happen, I remember it was fine. Maybe I rebased wrongly at some point.
Will fix now.